sklearn/model_selection/tests/test_validation.py

```diff
@@ -1450,3 +1450,25 @@ def test_fit_and_score():
     # check if the same warning is triggered
     assert_warns_message(FitFailedWarning, warning_message, _fit_and_score,
                          *fit_and_score_args, **fit_and_score_kwargs)
+
+    # check if exception was raised, with default error_score argument
+    assert_raise_message(ValueError, "Failing classifier failed as required",
+                         _fit_and_score, *fit_and_score_args)
+
+    # check if warning was raised, with default error_score argument
```

Comment:
Nitpick, but I think  "is raised"  in the present is more common (in all 3 comments): after all, it's "assert_raise_message" not "asssert_raised_message".